digitransit-component/packages/digitransit-component-autosuggest/src/index.js

```diff
@@ -269,7 +269,7 @@ class DTAutosuggest extends React.Component {
     mobileLabel: PropTypes.string,
     lock: PropTypes.func.isRequired,
     unlock: PropTypes.func.isRequired,
-    refPoint: PropTypes.object,
+    refPoint: PropTypes.string,
```

Comment:
One of the shared npm libraries changes here. In this situation, you must do the following:
- increase minor version in package.json of changed library (SEMVER numbering)
- update the dependency of every package.json referring to the changed library
- increase the version number referring to changed library in digitransit-component/packages/digitransit-component/package.json

In this case, cascading changes stop at autosuggest-panel library level. If you change a very low level library,  need to update dependencies above it can spread very wide. Every time  you touch a library in any way, version numbers must be increased. Unfortunately we do not have any automation for this laborious process.